### PR TITLE
chore(blog) update grafana

### PR DESCRIPTION
**Pull request by amyshwang**
- instructions and screenshot to be updated after the 7.1 release
- Minor editorial changes to update wording and reduce the number of links

### CODE REVIEW

File: blog/2020-10-19-grafana-tutorial.mdx [hunk 1]
```diff
@@ -1,45 +1,44 @@
 ---
-title: Build a monitoring dashboard with QuestDB and Grafana
+title: Build a monitoring dashboard with Grafana and QuestDB
```
Comment (javier) [inline blog/2020-10-19-grafana-tutorial.mdx:2]: maybe we can change to "Build a time-series monitoring dashboard with Grafana and QuestDB"? Or "Time-series monitoring with Grafana and QuestDB"? Asking as people might want to search for time-series monitoring and grafana but they might not know questdb

File: blog/2020-10-19-grafana-tutorial.mdx [hunk 2]
```diff
@@ -1,45 +1,44 @@
...
   Use Grafana with QuestDB to build a monitoring dashboard for visualization of
   time series data.
-keywords:
-  - timeseries
-  - questdb
-  - grafana
 tags: [tutorial, grafana]
-image: /img/blog/shared/og-grafana.png
+image: /img/blog/2020-10-19/banner.png
+slug: build-monitoring-dashboard-grafana-questdb
```
Comment (javier) [inline blog/2020-10-19-grafana-tutorial.mdx:12]: if we change the title, change the slug accordingly to include time-series

File: blog/2020-10-19-grafana-tutorial.mdx [hunk 3]
```diff
@@ -78,10 +74,15 @@ password:admin
 ### Start QuestDB
 
 The Docker version for QuestDB can be run exposing the port `8812` for the
-PostgreSQL connection and port `9000` for the web and REST interface:
+PostgreSQL connection and port `9000` for the web and REST interface. Similar to
+the Grafana conatiner, we add the `--add-host` parameter:
```
Comment (javier) [inline blog/2020-10-19-grafana-tutorial.mdx:78]: typo: container

File: docs/third-party-tools/grafana.md [hunk 4]
```diff
@@ -15,14 +15,13 @@ to Grafana via the [Postgres](/docs/reference/api/postgres) endpoint.
...
-We have discovered that Grafana 9.4.x has issues connecting to QuestDB. An
-[issue](https://github.com/grafana/grafana/issues/64376) has been raised with
-Grafana. We recommend using Grafana version to 9.3.8 until the issue is fixed.
+## Start Grafana
 
-:::
+You can start a grafana docker container with this command:
 
+```shell
+docker run -it --rm --name=grafana -p 3000:3000 grafana/grafana:latest
```
Comment (javier) [inline docs/third-party-tools/grafana.md:23]: Should we use grafana/grafana:latest or grafana/grafana-oss:latest ?

File: blog/2020-10-19-grafana-tutorial.mdx [hunk 5]
```diff
@@ -1,45 +1,44 @@
...
-connects to one or more data-sources to retrieve data, which is then visualized
-by the user in a browser.
+Grafana is an open-source platform for time-series analytics. It allows you to
+query, visualize, alert on, and understand your metrics no matter where they are
+stored. With its powerful query language, you can quickly analyze complex data
+sets and create dynamic dashboards to monitor your applications and
+infrastructure. Grafana also provides an ever-growing library of plugins for
+data sources, panel types, and visualizations.
+
+Grafana is a data visualization tool consisting of a server that connects to one
```
Comment (javier) [inline blog/2020-10-19-grafana-tutorial.mdx:39]: If we started this section saying "Grafana is..." I find it weird that we now say it is a different thing

**Review by javier — CHANGES_REQUESTED** [review]
Left several comments. I believe some can be ignored, but some need addressing

File: blog/2020-10-19-grafana-tutorial.mdx (hunk 1 above)
Comment (amyshwang) [reply blog/2020-10-19-grafana-tutorial.mdx:2]: I think "Time-Series Monitoring Dashboard with Grafana and QuestDB" would benefit from having "monitoring dashboard" keyword so I am using this!

File: blog/2020-10-19-grafana-tutorial.mdx [hunk 6]
```diff
@@ -162,9 +164,8 @@ builder in the bottom half:
   src="/img/blog/2020-10-19/blank-panel.png"
 />
 
-Toggle the query editor to **text edit mode** by clicking the pencil icon or by
-clicking the **Edit SQL** button. The query editor will now accept SQL
-statements that we can input directly:
+Toggle the query editor by using the builder or code panel. The query editor
```
[the remark this answers is not in the document]
Comment (javier) [reply blog/2020-10-19-grafana-tutorial.mdx]: Interesting. I see a different thing. You are probably using the grafana docker image, not the grafana-oss docker image. I would recommend using grafana-oss, as it is open source. The other is enterprise. I addressed this on a previous comment. This is what I see on oss

image


If I start with the grafana enterprise version, I see what you see

[the remark this answers is not in the document]
Comment (amyshwang) [reply blog/2020-10-19-grafana-tutorial.mdx]: I see! I have updated the screenshot and revert my changes.

File: blog/2023-04-12-grafana-tutorial.mdx [hunk 7]
```diff
@@ -117,30 +118,36 @@ In Grafana, select to the cog icon to expand the **Configuration** menu, select
...
-destructive to a dataset.
+Note that by default, Grafana does not validate that queries are read-only. This
+means it's possible to run queries such as `drop table x` in Grafana which would
+be destructive to a dataset.
 
-To protect against this, set a dedicated QuestDB instance **read-only mode** by
-setting the property `http.security.readonly=true` in your `server.conf`.
-Details of setting this configuration can be found on QuestDB's
-[configuration page](/docs/reference/configuration).
+To protect against this, we have set a dedicated QuestDB instance **read-only
```
Comment (javier) [inline blog/2023-04-12-grafana-tutorial.mdx:137]: This is not a dedicated read only mode instance anymore. It is enabling a read only user

**Review by javier — APPROVED** [review]
Looking great!

**Pull request merged**